packages/aws-rfdk/lib/lambdas/nodejs/lib/ecr-provider/ecr-provider.ts

```diff
@@ -0,0 +1,146 @@
+/**
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import * as fs from 'fs';
+import { IncomingMessage } from 'http';
+import * as https from 'https';
+
+/**
+ * Properties for constructing a {@link ThinkboxEcrProvider}
+ */
+export interface ThinkboxEcrProviderProperties {
+  /**
+   * The region of the desired ECR
+   */
+  readonly region: string;
+}
+
+/**
+ * The version provider parses a JSON file containing references to ECRs that serve Thinkbox's Deadline Docker images.
+ * It  can be downloaded or loaded from local file and returns the ECR ARN prefix for a given region.
+ */
+export class ThinkboxEcrProvider {
+  /**
+   * The URL to obtain the ECR index from.
+   */
+  private static readonly ECR_INDEX_URL = 'https://downloads.thinkboxsoftware.com/deadline_ecr.json';
```

Comment:
Just confirming... this URL hasn't changed at all, correct?

Reply:
That's right. This is URL where the ECR index will be.

Reply:
This is changing from `deadline_ecr.json` &rarr; `thinkbox_ecr.json`

Reply:
This is the correct URL - resolved